src/components/Book.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = props => {
+  const { book } = props;
+  const { id, title, category } = book;
+  return (
+    <tr>
+      <td>{id}</td>
+      <td>{title}</td>
+      <td>{category}</td>
+    </tr>
+  );
+};
+
+Book.defaultProps = {
+  book: {
+    id: '',
+    title: '',
+    category: '',
+  },
+};
```

Comment:
I dont think this is useful as you dont plan on having empty books, you will control it at level of the front-end interface FORMS

Reply:
It ain't but linter demands are linter demands.